packages/fileeditor-extension/src/index.ts

```diff
@@ -358,12 +358,16 @@ function activate(app: JupyterLab, restorer: ILayoutRestorer, editorServices: IE
 
   commands.addCommand(CommandIDs.markdownPreview, {
     execute: () => {
-      let path = tracker.currentWidget.context.path;
+      let widget = tracker.currentWidget;
+      if (!widget) {
+        return;
+      }
+      let path = widget.context.path;
       return commands.execute('markdownviewer:open', { path });
     },
     isVisible: () => {
       let widget = tracker.currentWidget;
-      return widget && PathExt.extname(widget.context.path) === '.md';
+      return widget && PathExt.extname(widget.context.path) === '.md' || false;
```

Comment:
Huh, strange. I guess my C is tripping me up, I didn't really see how `&&` could return anything other than a `boolean`.